docs/mutations.md

```diff
@@ -73,6 +73,11 @@ Do you have a suggestion for a (new) mutator? Feel free to create an [issue](htt
 |`new Collection<int> { 1, 2 };`           | `new Collection<int> { };`           |
 |`new Dictionary<int, int> { { 1, 1 } };`  | `new Dictionary<int, int> { };`      |
 
+## Object initialization (_initializer_)
```

Comment:
I think it's better to rename the other heading to `Initialization` and put this one under there as well. Because they are also in the same mutator class group

Reply:
Put them under the same heading 👍